projects/shared/src/lib/class/user.ts

```diff
@@ -117,23 +116,7 @@ export class User implements IUser {
    * @return boolean
    */
   get isPatron(): boolean {
-    return this._patronRoles.includes('patron');
-  }
-
-  /**
-   * Is the user a librarian
-   * @return boolean
-   */
-  get isLibrarian(): boolean {
-    return this._patronRoles.includes('librarian');
-  }
-
-  /**
-   * Is the user a system_librarian
-   * @return boolean
-   */
-  get isSystemLibrarian(): boolean {
-    return this._patronRoles.includes('system_librarian');
+    return this.patrons?.some((patron: IPatron) => 'patron' in patron);
```

Comment:
don't you have an enum class to list all PatronRole ? Should be better than use 'patron' string here